packages/@aws-cdk/aws-s3/test/test.bucket.ts

```diff
@@ -465,6 +467,17 @@ export = {
           "Export": {
             "Name": "S1:MyBucketDomainNameF76B9A7A"
           }
+        },
+        "MyBucketWebsiteURL9C222788": {
```

Comment:
I found the logical ID here (i.e. `MyBucketWebsiteURL9C222788`) from running the tests, watching them fail and observing the output.

I'm not sure if I was meant to have determined the output from somewhere else?